azure-pipelines/build-pr.yml

```
@@ -199,23 +198,19 @@ jobs:
           pytest_mark: after_training_hello_container
           test_run_title: tests_after_training_hello_container
 
-  # Run the Lung model. This is a large model requiring a docker image with large memory. This tests against
-  # regressions in AML when requesting more than the default amount of memory. This needs to run with all subjects to
-  # trigger the bug, total runtime 10min
-  - job: TrainLung
+  - job: TrainLungRegression
     dependsOn: CancelPreviousJobs
     variables:
       - name: model
-        value: 'Lung'
+        value: 'LungRegression'
       - name: tag
-        value: 'LungPR'
+        value: 'TrainLungRegressionModel'
       - name: more_switches
-        value: '--pl_deterministic --num_epochs=1 --feature_channels=16 --show_patch_sampling=0 --train_batch_size=4 --inference_on_val_set=False --inference_on_test_set=False '
+        value: '--log_level=DEBUG --pl_deterministic --use_dataset_mount=True --regression_test_folder=RegressionTestResults/PR_LungRegression --regression_test_csv_tolerance=0.001'
```

You have the `deterministic` flag already in the model itself.
```suggestion
        value: '--log_level=DEBUG --use_dataset_mount=True --regression_test_folder=RegressionTestResults/PR_LungRegression --regression_test_csv_tolerance=0.001'
```